doc/source/MANUAL.rst

```diff
@@ -205,6 +205,16 @@ in some cases printing these attachments on a successful tests is not the
 preferred behavior. You can use the ``--suppress-attachments`` flag to disable
 printing stdout or stderr attachments for successful tests.
 
+While by default attachments for captured stdout and stderr are printed, it
+is also possilbe that a test has other text attachments (a common example is
+python logging) which are not printed on successful test execution, only on
+failures. If you would like to have these attachments also printed for
+successful tests you can use the ``--all-attachments`` flag to print all text
+attachments on both successful and failed tests. If both ``--all-attachments``
+and ``--suppress-attachments`` are set then the ``--suppress--attachments``
+flag will take priority and no attachments will be printed for successful
```

Comment:
Wouldn't it make more sense to error in this case?

Reply:
I was just being lazy, doing it this way was slightly easier to implement. In my mind it doesn't matter exactly what the behavior is as long as we're explicit about how it works. That being said, this just made me realize there is a broken edge case with mixed inputs right now. If you set `suppress-attachments` in the user config file and `--all-attachments` on the cli that will mean no attachments print. So I'll have to change the behavior here anyway to make sure the cli takes precedence for both. I'll change this to an error at the same time since I'm not attached to any behavior.